### PR TITLE
Add Python3 support

### PR DESCRIPTION
**Pull request by ktbyers**
<!-- Make sure you have read http://napalm.readthedocs.io/en/latest/contributing/index.html --!>

### CODE REVIEW

**Comment by ktbyers** [comment]
Requires this fix to pyFG:

https://github.com/spotify/pyfg/pull/13

**Comment by dbarrosop** [comment]
I will ping Spotify again, if they don't answer I guess we will have to consider forking the project as we can't depend on an unmaintained library : (

**Comment by mirceaulinic** [comment]
Dafuq... @coveralls sent the message above (https://github.com/napalm-automation/napalm-fortios/pull/36#issuecomment-278533205) 28 days later?

**Comment by ktbyers** [comment]
Okay, this should be ready to go (if someone will review/approve).

File: test/unit/mocked_data/test_get_firewall_policies/normal/show_firewall_policy.txt [hunk 1]
```diff
@@ -9,15 +9,4 @@ config firewall policy
         set schedule "always"
         set service "ALL"
     next
-    edit 2
```
Comment (dbarrosop) [inline test/unit/mocked_data/test_get_firewall_policies/normal/show_firewall_policy.txt:12]: The expected_results.json was depending on the order of the dictionary. In other words, this test was failing on Python3 as the dictionary order changed.

Related to this, I think the get_firewall_policies() method is probably not implemented correctly. I will open a separate issue on this. I posted some information that @ebeahan was going to check (as I was making assumptions about Fortinet FW policies).

**Review by ebeahan — APPROVED** [review]
Thanks @ktbyers!

**Pull request merged**